### PR TITLE
Polyline

### PR DESCRIPTION
**Pull request by maulberto3**
Happy to PR Polyline functionality using your repo. You have no idea how happy I am about this accomplishment. Hope you find it useful, as much as I currently do for our app.
P.S. Will PR Polygon functionality too soon.
Sincerely, Mauricio.

### CODE REVIEW

**Comment by maulberto3** [comment]
Ok, so I fixed the small Travis issue by modifying the repo's .travis.yml file and make it use node version 8.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 5.0.0
+- 8.0.0
```
Comment (MicheleBertoli) [inline .travis.yml]: Can you please use `- "lts/*"`?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (MicheleBertoli) [inline .vscode/settings.json]: Can you please remove this file from the PR?

File: demo/index.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import React from 'react';
```
Comment (MicheleBertoli) [inline demo/index.js]: I would love to keep the demo as simple as possible (with just the events).
Do you mind reverting these changes?

File: src/components/gmaps.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
```
Comment (MicheleBertoli) [inline src/components/gmaps.js]: Unneeded empty line.

File: .travis.yml (hunk 1 above)
Comment (maulberto3) [reply .travis.yml]: Sure.

File: .vscode/settings.json (hunk 2 above)
Comment (maulberto3) [reply .vscode/settings.json]: Sure, removed.

File: demo/index.js (hunk 3 above)
Comment (maulberto3) [reply demo/index.js]: Oh sure, removed them.

File: src/components/gmaps.js (hunk 4 above)
Comment (maulberto3) [reply src/components/gmaps.js]: Sure, done.

File: demo/index.js (hunk 3 above)
Comment (maulberto3) [reply demo/index.js]: Oh sure, reverted as original.

**Comment by maulberto3** [comment]
So I had a bit of commit messages *mix-up*, but everything'd fine now.

**Comment by MicheleBertoli** [comment]
🙌Thank you very much, @maulberto3.

**Comment by maulberto3** [comment]
You are welcome @MicheleBertoli.

**Pull request merged**